heltour/tournament/views.py

```diff
@@ -1345,6 +1345,59 @@ def team(season):
                 season_score += bye.score()
                 season_score_total += 1
         season_perf_rating = season_perf.calculate()
+        
+        def season_performance(season):
```

Comment:
Most of the above code (the current season perf calculation) should be replaced with a call to this function (to avoid duplication).